### PR TITLE
FEV-423 Mange script for webpack build inside cli. Added relevant com…

### PR DESCRIPTION
**Pull request by alanovyy**
…mands.

### CODE REVIEW

**Comment by kaltura-hooks** [comment]
Hi @alanovyy,
Thank you for contributing this pull request!
Please sign the Kaltura CLA so we can review and merge your contribution.
Learn more at http://bit.ly/KalturaContrib

File: packages/playkit-js-scripts/bin/kcontrib.js [hunk 1]
```diff
@@ -25,13 +25,9 @@ let script = scriptIndex === -1 ? args[0] : args[scriptIndex];
 const nodeArgs = scriptIndex > 0 ? args.slice(0, scriptIndex) : [];
```
Comment (esakal) [inline packages/playkit-js-scripts/bin/kcontrib.js]: General comment:
The generated folder still contain the webpack files: 
1. webpack.common.js
2. webpack.dev.js
3. webpack.prod.js

Comment (esakal) [inline packages/playkit-js-scripts/bin/kcontrib.js]: General comment - `package.json` still contain dependencies of `webpack`

 "webpack": "4.41.2",
    "webpack-cli": "3.3.10",
    "webpack-dev-server": "3.9.0",
    "webpack-merge": "4.2.2"

source-map-loader
html-webpack-plugin
css-loader
clean-webpack-plugin
copy-webpack-plugin

etc...

Comment (alanovyy) [reply packages/playkit-js-scripts/bin/kcontrib.js]: I will remove them when serve command will be transfered to cli
FEV-439

Comment (alanovyy) [reply packages/playkit-js-scripts/bin/kcontrib.js]: I will remove them in FEV-439 when we gonna move all webpack-dependent stuff to cli

**Pull request merged**